packages/testlab/src/__tests__/unit/to-json.test.ts

```diff
@@ -62,6 +62,50 @@ describe('toJSON', () => {
     expectUndefined(value);
   });
 
+  it('handles `object | null`', () => {
+    const input: object | null = Math.random() ? {} : null;
```

Comment:
Wouldn't this always yield `{}`?

Reply:
> The Math.random() function returns a floating-point, pseudo-random number in the range 0–1 (inclusive of 0, but not 1) with approximately uniform distribution over that range

I think `Math.random() ?` returns `true` in most cases.

Reply:
Yes, so the behavior of `input = null` will rarely get tested.